builder/role_image.go

```diff
@@ -321,6 +324,13 @@ func (j roleBuildJob) Run() {
 		}
 	}
 
+	if j.builder.metricsPath != "" {
```

Comment:
Is it intentional that this doesn't output metrics when it doesn't have to build anything?